src/controllers/event/addOrganizer.ts

```diff
@@ -0,0 +1,57 @@
+import * as Interfaces from "@interfaces";
+import * as Errors from "@errors";
+import * as Utils from "@utils";
+import { prisma } from "@utils/prisma";
+import { EventOrganiser } from "@prisma/client";
+
+const addOrganizer: Interfaces.Controller.Async = async (req, res, next) => {
+  const { eventId: EID } = req.params;
+  const eventId = String(EID);
+
+  if (!eventId || eventId.length !== 24)
+    return next(Errors.Module.invalidInput);
+
+  const { organizers }: { organizers: [string] } = req.body;
+
+  if (!organizers) return next(Errors.Module.invalidInput);
+
+  if (!organizers.every((organizer) => organizer.length === 24)) {
+    return next(Errors.Module.invalidInput);
+  }
+
+  const results = await Promise.all(
+    organizers.map(async (organizer: string) => {
+      const user = await prisma.user.findFirst({ where: { id: organizer } });
+      if (!user) {
+        return false;
+      }
+      return true;
+    })
+  );
```

Comment:
Suggestion: I believe that I have seen this same piece of code in your previous changes [here](https://github.com/gdsc-nits-org/tecnoesis-api/blob/dev/src/controllers/event/add.ts#L96-L104), and to avoid this repeatation of code you can try making a util function [here ](https://github.com/gdsc-nits-org/tecnoesis-api/blob/dev/src/utils/event.ts)with the same funcationality and use that.

Reply:
implemented and changed in all the places.